app/scripts/metamask-controller.js

```diff
@@ -698,7 +702,7 @@ export default class MetamaskController extends EventEmitter {
         this.networkController.getProviderAndBlockTracker().provider,
       // NOTE: This option is inaccurately named; it should be called
       // onNetworkDidChange
```

Comment:
```suggestion
```

probably don't need this comment anymore

Reply:
🙏